src/estimate.coffee

```diff
@@ -88,16 +88,18 @@ estimateFor = ({ robot, res, ticketId, room }) ->
   else
     msg = "Median vote of #{median(ticket)} by #{listVoters(ticket, true)}"
 
-  if room then robot.messageRoom(room, msg) else res.send msg
-
   # post to pivotal tracker
   projectId = team?.projectId
   if HUBOT_PIVOTAL_TOKEN? && projectId?
+    messagePrefix = msg
     updatePivotalTicket({
-      robot, res, projectId, ticketId, points: median(ticket), room
+      robot, res, projectId, ticketId, points: median(ticket), room, messagePrefix
     })
+  else
+    if room then robot.messageRoom(room, msg) else res.send msg
+
 
-updatePivotalTicket = ({ robot, res, projectId, ticketId, points, room }) ->
+updatePivotalTicket = ({ robot, res, projectId, ticketId, points, room, messagePrefix }) ->
```

Comment:
the logic around mutating and reforming the message in this PR is getting a bit jenky and complex. There's likely room for a function that handles the message composition logic. Not a blocker but something to think about here